sprocs/sync_assessments.sql

```diff
@@ -385,6 +386,17 @@ BEGIN
                             computed_max_auto_points := (assessment_question->>'max_points')::double precision;
                         END IF;
                     END IF;
+
+                    IF (assessment_question->>'question_id')::bigint IS NULL THEN 
+                        -- TODO this should NEVER be hit in production, because it is inserting 
+                        -- fake questions into the question table to make syn succeed
+                        INSERT INTO questions AS dest (course_id, qid, uuid, deleted_at)
+                        VALUES (syncing_course_id, null, null, null) RETURNING dest.id INTO new_question_id;
+
+                    ELSE 
+                        new_question_id := (assessment_question->>'question_id')::bigint;
+                    END IF;
+
```

Comment:
@nwalters512 what are your thoughts on this chunk of code here? 

This is inserting dummy questions in place of shared questions so that we can succeed on assessment sync even we don't have access to the shared questions locally. 

Should we be adding some extra guardrails here to make sure this code never runs in production?